src/org/labkey/test/pages/core/admin/ShowAdminPage.java

```diff
@@ -141,6 +141,12 @@ public LookAndFeelSettingsPage clickLookAndFeelSettings()
         return new LookAndFeelSettingsPage(getDriver());
     }
 
+    public void clickResponseServerConfiguration()
```

Comment:
I guess there's precedent for `ShowAdminPage` knowing about all the module-specific links...